packages/dashboard/src/components/resizablePanes/index.tsx

```diff
@@ -45,6 +53,8 @@ export const ResizablePanes: FC<ResizablePanesProps> = ({ leftPane, centerPane,
   /** Current widths of the three panes, in px */
   const [leftPaneWidth, setLeftPaneWidth] = useState(DEFAULT_SIDE_PANE_WIDTH);
   const [rightPaneWidth, setRightPaneWidth] = useState(DEFAULT_SIDE_PANE_WIDTH);
```

Comment:
do we still want the pane to be resizable if it is also collapsible? If so, should we enforce some min / max standards?

Reply:
yes! there will be a follow up story for it